app/views/Profile/index.tsx

```diff
@@ -99,6 +99,9 @@ function Profile(props: Props) {
     } else if (userDetails?.userType === 'PUBLISHER') {
         profileDetails = userDetails?.publisher;
         nameLabel = strings.publisherNameLabel;
+    } else if (userDetails?.userType === 'INSTITUTIONAL_USER') {
+        profileDetails = userDetails?.publisher;
```

Comment:
this doesn't look correct. do we get profile details of the institution from `userDetails?.publisher`.